src/Entity/RoundResult.php

```diff
@@ -245,6 +255,18 @@ public function getResult()
     public function setResult($result): void
     {
         $this->result = $result;
+        switch ($result) {
```

Comment:
I would advice rather not map these two fields, but move this logics to the method getWhiteResult and getBlackResult. It's enough to store result code to database in order to transform it to white score and black score. Please remove the mapping, also you don't need to keep these fields. Just implement this calculation inside getWhiteResult and getBlackResult methods